conanfile.py

```diff
@@ -24,7 +24,7 @@
 
 class libhal_rmd_conan(ConanFile):
     name = "libhal-rmd"
-    version = "3.0.1"
+    version = "3.0.2"
```

Comment:
```suggestion
    version = "3.1.0"
```

Because you are adding a new feature that is available as a new API, you must bump the minor number up for semantic versioning.

Reply:
Ah, makes sense.